magpy/gui/magpy_gui.py

```diff
@@ -143,7 +143,18 @@ def saveini(optionsdict): #dbname=None, user=None, passwd=None, host=None, dirna
         optionsdict['fitdegree'] = '5'
     if optionsdict.get('fitknotstep','') == '':
         optionsdict['fitknotstep'] = '0.3'
-
+    if optionsdict.get('edgeid','') == '':
+        optionsdict['edgeid'] = ['BOU', 'BDT', 'TST', 'BRW', 'BRT', 'BSL',
```

Comment:
can users enter a custom observatory code into the combo box, or does it need to be in this list?